### DISCUSSION TITLE
High Availability mode for Ansible Semaphore deployment

### DISCUSSION THREAD

**Discussion by orez-fu** [semaphoreui/semaphore#1827] [Q&A] [answered]
Semaphore support runner to execute tasks from Semaphore Server. Whether could we have multiple Semaphore server in cluster? If we have 3 Semaphore server configure to the same MariaDB, and have Nginx as a load balancer to forward request to 3 Semaphore server, will it work without any trouble?

**Comment by tboerger** [comment, marked as answer]
The queue is currently built in-memory. This should rely on something external, otherwise it's not possible to run multiple instances of semaphore.
